### DISCUSSION TITLE
Keyboard Shortcuts

### DISCUSSION THREAD

**Discussion by JamesBranchFCPX** [CommandPost/FCPCafe#150] [Final Cut Pro Questions] [answered]
Trying to use keyboard commands more. How do you switch from the sidebar to the viewer and also activate clips in the viewer so they go from highlighted grey to blue - using the keyboard?

**Comment by florian-duffe** [comment]
I thought there is a clip from Steve Martin (Ripple Training) about it, but couldn't find it quickly. Gotta have look tomorrow!

**Reply by florian-duffe** [reply, marked as answer]
found it. pretty old, but it should work everything as described. https://www.youtube.com/watch?v=Proy2VTT03U

**Reply by JamesBranchFCPX** [reply]
Superb! Thanks. In Ripple Training we trust. So odd that video didn't come up in my YouTube searches.
